src/prefect/input/run_input.py

```diff
@@ -1,3 +1,64 @@
+"""
+This module offers a framework for data exchange between flow runs enabling
+real-time interaction and efficient data handling. It's designed to facilitate
+dynamic communication within distributed or microservices-oriented systems,
+making it ideal for scenarios requiring continuous data synchronization and
+processing.
+
+Use this module to create responsive, real-time applications where different
+flow runs need to communicate and exchange data dynamically. It's particularly
+useful for systems that require ongoing data input and output.
```

Comment:
nit: Maybe this could be more concise? It feels like these two paragraphs are saying similar things.